app/src/main/java/org/jellyfin/androidtv/preferences/ui/EditLongPreference.kt

```diff
@@ -0,0 +1,25 @@
+package org.jellyfin.androidtv.preferences.ui
+
+import android.content.Context
+import android.util.AttributeSet
+import androidx.preference.EditTextPreference
+import androidx.preference.PreferenceViewHolder
+
+class EditLongPreference : EditTextPreference {
```

Comment:
To make custom views work you need to pass 2 parameters to the super: the context and attributeset. The others are optional.
Context will never be null so you can remove the question mark from that one.
An example for the extra constructors that should be added can be found in the [Rating.kt] or [MultiBadgeImageCardView.kt] from the details branch.

[Rating.kt]: https://github.com/jellyfin/jellyfin-androidtv/blob/item-details-redesign/app/src/main/java/org/jellyfin/androidtv/ui/widget/Rating.kt#L15
[MultiBadgeImageCardView.kt]: https://github.com/jellyfin/jellyfin-androidtv/blob/item-details-redesign/app/src/main/java/org/jellyfin/androidtv/ui/MultiBadgeImageCardView.kt